jaeger-extension-ballerina/build.gradle

```diff
@@ -36,11 +36,17 @@ dependencies {
 
     externalJars "io.jaegertracing:jaeger-core:${jaegerVersion}"
     externalJars "io.jaegertracing:jaeger-thrift:${jaegerVersion}"
-    externalJars "io.opentracing:opentracing-util:${openTracingVersion}"
-    externalJars "io.opentracing:opentracing-noop:${openTracingVersion}"
+    externalJars "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}"
+    externalJars "io.opentelemetry:opentelemetry-api-metrics:${openTelemetrySemconvVersion}"
```

Comment:
Do we need the metrics api ?

Reply:
Yes, `BatchSpanProcessor` needs `GlobalMetricsProvider` which resides in api-metrics.